src/main/test/java/com/zufar/onlinestore/product/util/ProductPaginationDefaults.java

```diff
@@ -0,0 +1,26 @@
+package com.zufar.onlinestore.product.util;
+
+import lombok.Getter;
+
+@Getter
+public enum ProductPaginationDefaults {
```

Comment:
For me it looks strange to store Defaults for ProductPagination in enum. Is it really best practise or there is better solutions for storing Defaults?
It just looks bulky.

Reply:
fixed